cli/scripts/build.ts

```diff
@@ -58,20 +58,20 @@ function preparePackageForNpmRelease (json: any, branchName?: string): any {
   return json
 }
 
-function makeUserPackageFile (branchName?: string): Promise<any> {
-  return fs.readJsonAsync(packageJsonSrc)
-  .then((json: any) => preparePackageForNpmRelease(json, branchName))
-  .then((json: any) => {
-    return fs.outputJsonAsync(packageJsonDest, json, {
-      spaces: 2,
-    })
-    .return(json) // returning package json object makes it easy to test
-  })
+async function makeUserPackageFile (branchName?: string): Promise<any> {
+  const json = await fs.readJson(packageJsonSrc)
+  const jsonPrepared = preparePackageForNpmRelease(json, branchName)
+
+  await fs.outputJson(packageJsonDest, jsonPrepared, {
+    spaces: 2,
+  }) // returning package json object makes it easy to test
+
+  return jsonPrepared
 }
 
 export default makeUserPackageFile
 
-if (!module.parent) {
+if (require.main === module) {
```

Comment:
module.parent is deprecated. we likely don't need the check here since this is always invoked as a script, unless we are unit testing.